infra/modules/providers/azure/app-service/main.tf

```diff
@@ -37,18 +37,19 @@ resource "azurerm_app_service" "appsvc" {
   }
 
   auth_settings {
-    enabled          = local.app_configs[count.index].ad_client_id == "" ? false : true
+    enabled          = false
     issuer           = format("https://sts.windows.net/%s", local.tenant_id)
     default_provider = "AzureActiveDirectory"
     active_directory {
-      client_id = local.app_configs[count.index].ad_client_id
+      client_id = ""
     }
   }
 
   site_config {
     linux_fx_version     = format("DOCKER|%s/%s", var.docker_registry_server_url, local.app_configs[count.index].image)
     always_on            = var.site_config_always_on
     virtual_network_name = var.vnet_name
+    scm_type             = "None"
```

Comment:
What's the intent behind adding this change to the site_config?

Reply:
I thought not having this set to "none" caused some local issues I was having that displayed username/password information in the terraform plan. Turns out, its not an issue in the pipeline. Since this defaults to "none", I removed this line after all.